kolibri/core/lessons/serializers.py

```diff
@@ -127,7 +127,7 @@ def update(self, instance, validated_data):
         # Update the scalar fields
         instance.title = validated_data.get("title", instance.title)
         instance.description = validated_data.get("description", instance.description)
-        instance.is_active = validated_data.get("active", instance.is_active)
+        instance.is_active = validated_data.get("is_active", instance.is_active)
```

Comment:
Looking above, in the create method, validated data is just passed directly to the Lesson.objects.create method, so it does make sense that the validated data contains the attribute mapped to source.